### PR TITLE
BTS-818: byebye users test

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

convert one more test

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/20543/ green. Due to editing the subject jenkins won't find this :/

File: tests/js/client/shell/shell-api-users.js [hunk 1]
```diff
@@ -0,0 +1,793 @@
...
+  return {
+    setUp: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i);;
+      }
+    },
+
+    tearDown: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i.to_s);;
```
Comment (jsteemann) [inline tests/js/client/shell/shell-api-users.js:50]: `to_s` does not exist in JavaScript
```suggestion
        arango.DELETE_RAW("/_api/user/users-" + i);
```

File: tests/js/client/shell/shell-api-users.js [hunk 2]
```diff
@@ -0,0 +1,793 @@
...
+const contentType = forceJson ? "application/json" :  "application/x-velocypack";
+const jsunity = require("jsunity");
+
+let api = "/_api/user";
+
+function user_managementSuite () {
+  return {
+    setUp: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i);;
```
Comment (jsteemann) [inline tests/js/client/shell/shell-api-users.js:44]:
```suggestion
        arango.DELETE_RAW("/_api/user/users-" + i);
```

File: tests/js/client/shell/shell-api-users.js [hunk 3]
```diff
@@ -0,0 +1,793 @@
...
+      }
+    },
+
+    tearDown: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i.to_s);;
```
Comment (jsteemann) [inline tests/js/client/shell/shell-api-users.js:84]:
```suggestion
        arango.DELETE_RAW("/_api/user/users-" + i);
```

**Pull request closed without merging**